### ISSUE TITLE
 Argument 4 passed to PhpMyAdmin\\Export::getFilenameAndMimetype() must be of the type string, boolean given

### ISSUE THREAD

**Issue by barkermn01** [phpmyadmin/phpmyadmin#14323]
### Steps to reproduce
1.Export a Table or Database

### Expected behaviour
Should produce the Export file.

### Actual behaviour
Error 500, 
[Thu May 24 09:23:19.062491 2018] [proxy_fcgi:error] [pid 7936] [client 172.31.41.184:51750] AH01071: Got error 'PHP message: PHP Fatal error:  Uncaught TypeError: Argument 4 passed to PhpMyAdmin\\Export::getFilenameAndMimetype() must be of the type string, boolean given, called in /var/www/**OMITED**/phpMyAdmin/export.php on line 355 and defined in /var/www/**OMITED**/phpMyAdmin/libraries/classes/Export.php:274\nStack trace:\n#0 /var/www/**OMITED**/phpMyAdmin/export.php(355): PhpMyAdmin\\Export->getFilenameAndMimetype('table', 'on', Object(PhpMyAdmin\\Plugins\\Export\\ExportSql), false, '@TABLE@')\n#1 {main}\n  thrown in /var/www/**OMITED**/phpMyAdmin/libraries/classes/Export.php on line 274\n'

### Server configuration
**Operating system**:
Ubuntu 16.04

**Web server:**
Apache 

**Database:**
MariaDB 10

**PHP version:**
PHP 7.2 - FPM

**phpMyAdmin version:**
Version information: 5.0.0-dev, latest stable version: 4.8.0.1
Git revision: c037e68 from master branch,
committed on May 23, 2018 at 08:34 PM by Weblate, 
authored on May 23, 2018 at 10:33 PM by Michal Čihař

### Client configuration
**Browser:**
Chrome Latest

**Operating system:**
Windows 10 Latest

**Comment by barkermn01** [comment]
As another note, this also means the Travis CI built tester is not set up correctly as this error should never get through. Travis Says the build has passed someone needs to add unit tests for exporting a table &/ Database.

**Comment by barkermn01** [comment]
Fixed by changing /export.php line 220

was 
```
$compression = false;
```

Changed it to 
```
$compression = isset($_REQUEST['compression'])? $_REQUEST['compression']:'gzip';
```

**Comment by ibennetch** [comment]
This is only present in master.

**Comment by ibennetch** [comment]
Yes, some of the testing is incomplete; we'd welcome a pull request to add testing for this aspect (or any others).

**Comment by ibennetch** [comment]
This should be fixed with 2a34c018633764ab0149d453dd2c8bfc8e79396f.

Thank you for your report.
